src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable no-nested-ternary */
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <div className="App">
+      <section className="Person">
+        <h2 className="Person__name">My name is {name}</h2>
+
+        {age && <p className="Person__age">I am {age}</p>}
+
+        <p className="Person__partner">
+          {isMarried
+            ? sex === 'm'
+              ? `${partnerName} is my wife`
+              : `${partnerName} is my husband`
```

Comment:
you can save nested (second level) ternary results to a variable to avoid ternary nesting

Reply:
thank you, i'm try